proto/api/workload/workload.proto

```diff
@@ -42,7 +44,51 @@ message X509SVID {
     repeated string federates_with = 5;
 }
 
+
+message JWTASVID {
+    string spiffe_id = 1;
+
+    // Encoded using JWS Compact Serialization
+    string svid = 2;
+}
+
+message JWTASVIDRequest {
+    repeated string audience = 1;
+
+    // SPIFFE ID of the JWT being requested
+    // If not set, all IDs will be returned
+    string spiffe_id = 2;
+}
+
+message JWTASVIDResponse {
+    repeated JWTASVID svids = 1;
+}
+
+message JWTABundlesRequest { }
+
+message JWTABundlesResponse {
+    // JWK sets, keyed by trust domain URI
+    map<string, bytes> bundles = 1;
+}
+
+message ValidateJWTASVIDRequest {
+    repeated string audience = 1;
```

Comment:
One scenario I can think of is migration of audience value. You'd want a window where the old audience value is still acceptable until all of the JWTs minted against the old audience expired.

Reply:
Hmm yea... It can be done on the other side too, i.e. the subject sends a token with both old and new audiences, and the migrating validator identifies with one or the other.

I'm having a hard time forming an opinion here, but it kind of feels like it's more natural to have the validator identify with a single subject... I'm curious to hear other thoughts on the matter though

Reply:
I guess my point is that the subject would have to be updated to know about the new audience value. I can imagine a partial rollout where old subject's are still requesting the old value, updated subject's are requesting the new value, and the target would be receiving JWT's from both.

Reply:
After some offline discussion, we've decided to move to a non-repeated audience field for now. It is much easier to transition to a repeated field later if we determine we need it than moving in the opposite direction.